src/main/scala/main/solutions/MazeSolver/Solver.scala

```diff
@@ -0,0 +1,33 @@
+package main.solutions.MazeSolver
+
+import scala.collection.mutable.{HashSet, Queue}
+
+object Solver {
+  def hasPathBFS(maze: Array[Array[Int]], start: Point, end: Point): Boolean = {
+    hasPathBFSHelper(maze, start, end, HashSet[Point]())
```

Comment:
what's the value of it? if it's not a recursive function, you don't need the helper. just instantiate the maop in the beginning of the function

Reply:
The signature of the `hasPathBFS` is without the visited set, so I created new function with another param.